doc/Known_issues.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: "Known issues"
 output: github_document
+editor:
+  markdown:
+    wrap:      80
+    canonical: true
```

Comment:
The default markdown configuration in the project [was changed to "sentence wrapping"](https://github.com/DaniMori/git-workshop/blob/87fd01c1130012a89f955d22fb9b64525f4f984d/rstudio_project.Rproj#L19), but this file was still using column wrapping instead.

I added the column wrapping configuration and reverted the "sentence wrapping" changes. Please **be mindful about what you add, and try not to bulk commit!** 😉